projects/ng-dynamic-breadcrumb/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "ng-dynamic-breadcrumb",
-  "description": "Angular 12 Dynamic Breadcrumb",
```

Comment:
Maybe better to keep version 7 for Angular 13 compatibility?  Or could version 7 cover 13 and 14? Or perhaps version 6 could be extended to cover 12,13 and 14 - I am not sure. There is possibility to to     "ng-packagr": ">=12.1.1 <15", etc